pc_helper_online.py

```diff
@@ -9,35 +12,43 @@ def main():
     """
     This code is used only for testing purpose.
     Allowing to prove that Public Cloud related logic is actually working without executing
-    a lot of code which is unrelated to pc_helper
+    a lot of code which is unrelated to pc_helper.
+    As input it getting same folder as bot-ng but processing only two variables which are related to Public Cloud
     """
     log = create_logger("pc_helper_online")
-    settings = {
-        "PUBLIC_CLOUD_TOOLS_IMAGE_QUERY": "https://openqa.suse.de/group_overview/276.json"
-    }
-    log.info("Calling apply_pc_tools_image with :\n%s", settings)
-    apply_pc_tools_image(settings)
-    log.info("Setting after the call to apply_pc_tools_image:\n%s", settings)
-
-    settings = {
-        "PUBLIC_CLOUD_PINT_QUERY": "https://susepubliccloudinfo.suse.com/v1/amazon/images/",
-        "PUBLIC_CLOUD_PINT_NAME": "suse-sles-12-sp5-v[0-9]{8}-hvm-ssd-x86_64",
-        "PUBLIC_CLOUD_PINT_REGION": "us-east-1",
-        "PUBLIC_CLOUD_PINT_FIELD": "id",
-    }
-    log.info("Calling apply_publiccloud_pint_image with :\n%s", settings)
-    apply_publiccloud_pint_image(settings)
-    log.info("Setting after the call to apply_publiccloud_pint_image:\n%s", settings)
-
-    settings = {
-        "PUBLIC_CLOUD_PINT_QUERY": "https://susepubliccloudinfo.suse.com/v1/amazon/images/",
-        "PUBLIC_CLOUD_PINT_NAME": "suse-sles-15-sp2-chost-byos-v[0-9]{8}-hvm-ssd-x86_64",
-        "PUBLIC_CLOUD_PINT_REGION": "us-east-1",
-        "PUBLIC_CLOUD_PINT_FIELD": "id",
-    }
-    log.info("Calling apply_publiccloud_pint_image with :\n%s", settings)
-    apply_publiccloud_pint_image(settings)
-    log.info("Setting after the call to apply_publiccloud_pint_image:\n%s", settings)
+    parser = ArgumentParser(
+        prog="pc_helper_online",
+        description="Dummy code to test functionality related to pc_helper code",
+    )
+    parser.add_argument(
+        "-c",
+        "--configs",
+        type=Path,
+        default=Path("/etc/openqabot"),
+        help="Directory with openqabot configuration metadata",
+    )
+    args = parser.parse_args()
+    loader = YAML(typ="safe")
+    log.info(f"Parsing configuration files from {args.configs}")
+    for p in Path(args.configs).glob("*.yml"):
+        try:
+            data = loader.load(p)
+            log.info(f"Processing {p}")
```

Comment:
```suggestion
            log.info(f"Processing {p}")
            data = loader.load(p)
```
I would log first in case `loader.load()` throws an exception but it's a matter of choice anyway.

Reply:
done